packaging/debian/build_debian.sh

```diff
@@ -27,11 +27,28 @@ build_root="${2:-$(mktemp -d)}"
 
 echo "Building $mode mode in $build_root"
 
+build_electron() {
+  echo "building Electron client"
+  # Remove the line below once this branch is in master.
+  git checkout cjb/DESKTOP-114-linux-electron-2
+  # Can't seem to get the right packages installed under NODE_ENV=production.
+  export NODE_ENV=development
+  backto=`pwd`
+  cd ../../
+  cd react-native
+  npm i
+  cd ../desktop
+  npm i
+  export NODE_ENV=production
+  cd $backto
```

Comment:
Instead of stashing the CWD, can we just run these commands inside a `(...)` subshell?